mpmath/functions/zeta.py

```diff
@@ -288,20 +288,18 @@ def terms():
 
 @defun_wrapped
 def bernpoly(ctx, n, z):
-    # Slow implementation:
-    #return sum(ctx.binomial(n,k)*ctx.bernoulli(k)*z**(n-k) for k in xrange(0,n+1))
```

Comment:
I think, this could be fixed, instead of silent removal.  `(-1)**k` factor, I guess.